Script/AtomicEditor/ui/frames/menus/HierarchyFrameMenu.ts

```diff
@@ -110,7 +115,8 @@ class HierarchyFrameMenus extends Atomic.ScriptObject {
                 node.scene.sendEvent("SceneEditNodeCreated", { node: newnode });
             }
 
-            return true;
+            // Let plugins handle context
+            return ServiceLocator.uiServices.hierarchyContextItemClicked(target, refid);
```

Comment:
I know I'm coming in late to the PR, but would it make more sense to pass the ```node``` attached to the target instead of the UIWidget?  I would think that most plugins would be interested in that node and not the menu item.  Also, in the other menu handler, if the menu was already handled, I didn't pass it on to the service locator.